src/app/components/forms/input/radio/terra-radio-group.component.spec.ts

```diff
@@ -0,0 +1,131 @@
+import {
+    async,
+    ComponentFixture,
+    TestBed
+} from '@angular/core/testing';
+import { TerraRadioInputComponent } from './terra-radio-input.component';
+import { TerraRadioGroupComponent } from './terra-radio-group.component';
+import { By } from '@angular/platform-browser';
+import { Component } from '@angular/core';
+import Spy = jasmine.Spy;
+
+
+@Component({
+    template: `<terra-radio-group name="radio">
+        <terra-radio-input [label]="'Option 1'" [value]="1"></terra-radio-input>
+        <terra-radio-input [label]="'Option 2'" [value]="2"></terra-radio-input>
+    </terra-radio-group>`
+})
+class RadioGroupTestComponent {}
+
+
+describe(`TerraRadioGroupComponent:`, () =>
+{
+    let component:RadioGroupTestComponent;
+    let fixture:ComponentFixture<RadioGroupTestComponent>;
+    let radioGroupComponent:TerraRadioGroupComponent;
+    beforeEach(async(() =>
+    {
+        TestBed.configureTestingModule({
+            declarations: [TerraRadioInputComponent, TerraRadioGroupComponent, RadioGroupTestComponent]
+        }).compileComponents();
+    }));
+
+    beforeEach(() =>
+    {
+        fixture = TestBed.createComponent(RadioGroupTestComponent);
+        component = fixture.componentInstance;
+    });
+
+    describe(`itself`, () =>
+    {
+        beforeEach(() =>
+        {
+            radioGroupComponent = TestBed.createComponent(TerraRadioGroupComponent).componentInstance;
+        });
+
+        it(`should create`, () =>
+        {
+            expect(component).toBeTruthy();
+            expect(radioGroupComponent).toBeTruthy();
+        });
+
+        it(`should initialize its inputs`, () =>
+        {
+            expect(radioGroupComponent.name).toBeUndefined();
+            expect(radioGroupComponent.legend).toBeUndefined();
+        });
+
+        // TODO: make it run..
+        xit(`should display given #legend text in the <legend>-element`, () =>
+        {
+            let legendElement:HTMLLegendElement = fixture.debugElement.query(By.css('legend')).nativeElement;
+            expect(legendElement.innerText).toBe('');
+
+            const legend:string = 'Legend';
+            radioGroupComponent.legend = legend;
+            fixture.detectChanges();
+
+            expect(legendElement.innerText).toBe(legend); // this fails.. why??
+        });
+
+        it(`setting the #value should update the #value`, () =>
+        {
+            const testValue:any = {test: true};
+            radioGroupComponent.value = testValue;
+            expect(radioGroupComponent.value).toBe(testValue);
```

Comment:
der test macht iwie keinen Sinn

Reply:
man müsste in dem radio button dann gucken ob der, welche gecheckt ist auch dem Wert entspricht...

Reply:
und das machst du ja in Zeile 116.

Reply:
Hier teste ich nur den Setter und den Getter, ob sie auch tun was ich erwarte :-P